_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -70,8 +70,9 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 {% highlight bash %}
 sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
-sudo yum -y install foreman-release-scl
+sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/centos-release-scl-rh-2-3.el7.centos.noarch.rpm
```

Comment:
I'd suggest to make it `sudo yum -y install centos-release-scl-rh` since this correctly uses GPG signatures. Especially with repo sources this is important. (granted, we don't sign plugins, but let's ignore that for now).

Then add a note that Scientific Linux and Oracle Linux need special instructions. Perhaps we should even make this section CentOS 7 instructions rather than EL 7 instructions.

Reply:
I considered that. We will need to technically add instructions to ensure CentOS extras repository is enabled to install centos-release-scl-rh directly. Sound good to add that as well?

Reply:
extras is enabled by default, but a note is not bad.